### DISCUSSION TITLE
Is there a way to load the cached data and re-validate later?

### DISCUSSION THREAD

**Discussion by saturnonearth** [Kong/swrv#125] [Q&A]
I am having an issue where it seems like every time I visit the page where SWRV is called it is re-validating the data, causing a significant delay before anything is displayed. I am still trying to figure out why there is a long delay, but I thought it might be because it is not using the already cached data immediately.

How do I load the already cached data (if there is some), and then update after if the validation comes back with any differences?

**Comment by saturnonearth** [comment]
> I am having an issue where it seems like every time I visit the page where SWRV is called it is re-validating the data, causing a significant delay before anything is displayed. I am still trying to figure out why there is a long delay, but I thought it might be because it is not using the already cached data immediately.
> 
> How do I load the already cached data (if there is some), and then update after if the validation comes back with any differences?

Nevermind! I was not using <n-link> with Nuxt, so it was actually reloading the page each time. I am still trying to figure out why there is a significant delay in the first load though..._that_, I do not understand, and have _no idea_ if it is SWRV related (probably not) or something else blocking the main paint.

**Reply by darrenjennings** [reply]
@saturnonearth if you have an example repo or codesandbox I can take a look.

**Reply by saturnonearth** [reply]
> @saturnonearth if you have an example repo or codesandbox I can take a look.

Unfortunately, the repo is private and is too complex at this point to create a codesandbox version. So I'm on my own here. But thank you!
